config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-main-periodics.yaml

```diff
@@ -116,10 +116,10 @@ periodics:
       resources:
         requests:
           cpu: "4000m"
```

Comment:
Should we set limits on some more jobs? I think a bunch of them only have requests (including upgrade jobs)

Reply:
AFAIK the jobs without limits are only the ones still running on GCP.
I would not invest time in optimizing those jobs because we have to migrate them soonish and also because AFAIK we don't have the same monitoring data that we get for jobs running in EKS

Reply:
ack, we have no monitoring data for these.

Reply:
Ah kk. Missed that we don't have data